docs/00_install/01_build-from-source/index.md

```diff
@@ -2,123 +2,19 @@
 content_title: Build Antelope from Source
 ---
 
-The shell scripts previously recommended for building the software have been removed in favor of a build process entirely driven by CMake. Those wishing to build from source are now responsible for installing the necessary dependencies. The list of dependencies and the recommended build procedure are in the README.md file. Instructions are also included for efficiently running the tests.
+The shell scripts previously recommended for building the software have been removed in favor of a build process entirely driven by CMake. Those wishing to build from source are now responsible for installing the necessary dependencies. The list of dependencies and the recommended build procedure are in the [`README.md`](https://github.com/AntelopeIO/leap/blob/release%2f3.1/README.md) file. Instructions are also included for efficiently running the tests.
```

Comment:
Btw, any reason for the "%2F" hex character in the branch name, instead of the explicit slash "/"?

Reply:
can you please use / instead of %2f? it makes it easier to read for the brave soul adventuring to read the md file content

Reply:
This is called [URL percent encoding](https://en.wikipedia.org/wiki/Percent-encoding). Here, `%2f` translates to a forward slash (`/`). The purpose of this encoding is to pass characters to a server in a URI that are otherwise not valid.

For example, in this case, we have a protocol (`https://`) and fully qualified domain name (`github.com.`) followed by a filesystem path on the server. The path is:
```js
/${ORGANIZATION}/${REPO}/blob/${BRANCH}/${FILENAME}
```
The branch is `release/3.1` which also contains a forward slash. If we include that forward slash in the URL unmodified, we are telling the server to go into the following folders.
```
AntelopeIO > leap > blob > release > 3.1 > README.md
```
With percent encoding, we are telling the server to go to this path.
```
AntelopeIO > leap > blob > release/3.1 > README.md
```
This conceptually makes more sense but, after doing some reading, I see that `git` does actually create a tree structure for references so it is probably stored this way on the server too.

![2022-10-21 11-43-05 - git refs heads folder structure](https://user-images.githubusercontent.com/34947245/197236035-5b4f0330-0106-4b7c-aa91-3594f9d57d68.png)

I can make them literal forward slashes (`/`).

Reply:
Done.